Rebuild-Black-Business/RBB-Website
# Feature/zip to latlng util

# Describe your PR

Returns a lat lng coordinate pair to help with Algolia radius searches.

Related to #
Fixes #

## Pages/Interfaces that will change

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

## Steps to test

1. Navigate to /businesses or /allies
2. Type a zip code, city name, place name, pretty much anything into the "Location" or "Zip" input
3. Check the console. You should see a lat / lng pair.
